apps/reference/docs/guides/platform/backups.mdx

```diff
@@ -0,0 +1,102 @@
+
+<aside>
+^ Database backups do not include objects stored via the Storage API, as the database only includes metadata about these objects. Restoring an old backup does not restore objects that have been deleted since then.
+
+</aside>
+
+---
+
+## Daily Backups
+
+All Supabase projects are backed up automatically on a daily basis. Access to these backups however are limited to Pro and Enterprise tier projects only. In terms of Recovery Point Objective (RPO), Daily Backups would be suitable for projects willing to lose up to 24 hours worth of data if disaster hits at the most inopportune time. If a lower RPO is required, enabling Point-in-Time Recovery should be considered.
+
+### Backup Process
+
+The PostgreSQL utility [pg_dumpall](https://www.postgresql.org/docs/current/app-pg-dumpall.html) is used to perform daily backups. An SQL file is generated, zipped up, and sent to our storage servers for safe keeping. 
+
+```
+Insert screenshot of Daily Backups dashboard
+```
+
+Daily backups can be accessed in the dashboard through `Database > Backups > Scheduled` or through the URL: `https://app.supabase.com/project/${ref}/database/backups/scheduled`. Pro tier projects can access the last 7 days’ worth of daily backups while Enterprise tier projects can access up to 30 days’ worth of daily backups. Users can restore their project to any one of the backups or download them as a zipped SQL file.
+
+### Restoration Process
+
+When selecting a backup to restore to, select the closest available one made before the desired point in time to restore to. Earlier backups can always be chosen too but do consider the number of days’ worth of data that could be lost. 
+
+```
+Insert screenshot of the Dashboard with the confirmation modal 
+```
+
+The Dashboard will then prompt for a confirmation before proceeding with the restoration. The project will be inaccessible following this. As such, do ensure to allot downtime beforehand. This is dependent on the size of the database. The larger it is, the longer the downtime will be. Once the confirmation has been given, the underlying SQL of the chosen backup is then run against the project. The PostgreSQL utility [psql](https://www.postgresql.org/docs/current/app-psql.html) is used to facilitate the restoration.
+
+```
+Insert screenshot of the Dashboard of the project in restoration mode
+```
+
+As the restoration occurs, the project on the Dashboard will be inaccessible as well. The Dashboard will display a notification once the restoration completes.
+
+```
+Insert screenshot of the Dashboard of the project completing restoration
+```
+
+---
```

Comment:
style nit: I don't think we need horizontal lines (kind of signifies it should be a separate doc/page)
```suggestion
```